CHANGES.txt

```diff
@@ -9,6 +9,9 @@ NOTE: 4.3.0 now requires Python 3.6.0 and above. Python 3.5.x is no longer suppo
 
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
+  From Prabhu S. Khalsa:
```

Comment:
Additions to this section should listed in alphabetical order by surname(last name). Can you make that change?

Reply:
Thanks, I thought maybe the latest was supposed to go on top. I'll change it now.